packages/caravan-health/src/spendType.ts

```diff
@@ -75,3 +76,43 @@ export function getSpendTypeScore(
       throw new Error("Invalid spend type");
   }
 }
+
+export function getInputWeight(
+  scriptType: MultisigAddressType,
+  config: { requiredSignerCount: number; totalSignerCount: number },
+): number {
```

Comment:
I think this is a bit cleaner. 
```suggestion
  { 
      requiredSignerCount,
      totalSignerCount
      scriptType
  }: { 
    requiredSignerCount: number; 
    totalSignerCount: number; 
    scriptType: MultisigAddressType 
}): number {
```